com.unity.render-pipelines.high-definition/Documentation~/Decal-Projector.md

```diff
@@ -45,6 +45,9 @@ Using the Inspector allows you to change all of the Decal Projector properties,
 
 ## Limitations
 
+- Decal Projector can affect Opaque Material with both a [Decal Shader](Decal-Shader.md) or a [Master Node Decal](Master-Node-Decal.md). It can affect Transparent Material only with a Decal shader. 
```

Comment:
- The Decal Projector can affect opaque Materials with either a [Decal Shader](Decal-Shader.md) or a [Decal master node](Master-Node-Decal.md). However, it can only affect transparent Materials with the [Decal Shader](Decal-Shader.md).